README.md

```diff
@@ -4,64 +4,64 @@
 [![codecov](https://codecov.io/gh/canonical/ubuntu-image/branch/main/graph/badge.svg?token=F9jE9HKo1a)](https://codecov.io/gh/canonical/ubuntu-image)
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/ubuntu-image)](https://goreportcard.com/report/github.com/canonical/ubuntu-image)
 
+This tool is used to build Ubuntu images:
 
-This tool is used to build Ubuntu images.  Currently builds Ubuntu Core snap-based
-images from model assertions and Ubuntu classic preinstalled images using image definition
-files, but it will be generalized to build more (eventually all) Ubuntu images.
+* Ubuntu Core snap-based images from model assertions
+* Ubuntu classic preinstalled images using image definition
 
+Future versions should be generalized to build more (eventually all) Ubuntu images.
 
-# Requirements
 
-Ubuntu 18.04 (Bionic Beaver) is the minimum platform requirement, but Ubuntu
-20.04 (Focal Fossa) or newer is recommended. All required third party packages are available in the
-Ubuntu archive.
+## Requirements
 
-If you want to run the test suite locally, you should install all the build
-dependencies named in the `debian/control` file.  The easiest way to do that
-is to run::
+Ubuntu 18.04 (Bionic Beaver) is the minimum platform requirement, but Ubuntu 20.04 (Focal Fossa) or newer is recommended. All required third party packages are available in the Ubuntu archive.
+
+To run the test suite locally, install all the build dependencies named in the `debian/control` file. Run the following command from the root directory of the repository:
```

Comment:
This instruction should probably be replaced because we do not package u-i as a deb. So I will soon submit a PR to remove this file once we found a proper way to keep the changelog somewhere. We also have instructions for contributors in HACKING.md. Maybe we should only gather this in a single place?

Reply:
> This instruction should probably be replaced because we do not package u-i as a deb. 

Of course. I keep forgetting.

> We also have instructions for contributors in HACKING.md. Maybe we should only gather this in a single place?

Actually, I was thinking of renaming HACKING.md to CONTRIBUTING.md, which seems to be the norm on GH these days. But I believe it makes sense to have both the README (for users) and a separate file for devs/contributors.

Reply:
I agree with the renaming and with separating the doc for users and for contributors.